tracer/src/Datadog.Trace/Agent/AgentWriter.cs

```diff
@@ -114,7 +108,13 @@ internal AgentWriter(IApi api, IStatsAggregator statsAggregator, IDogStatsd stat
 
         public Task<bool> Ping()
         {
-            return _api.SendTracesAsync(EmptyPayload, 0, false, 0, 0);
+            if (_emptyPayload.Array == null)
+            {
+                var data = Vendors.MessagePack.MessagePackSerializer.Serialize(Array.Empty<Span[]>());
```

Comment:
This is always `{ 0x90 }`, no need to call the serializer.

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/3856/commits/09296db9c5b1a8bf3d1c50f1740c5e02bc4ed003